web/helpers/filter.helper.ts

```diff
@@ -26,6 +27,24 @@ export const calculateTotalFilters = (filters: TFilters): number =>
         .reduce((curr, prev) => curr + prev, 0)
     : 0;
 
+/**
```

Comment:
We can use the existing function above using typescript generics. Please create an issue for that and we will track it as a separate change.